tests/factories/__init__.py

```diff
@@ -0,0 +1 @@
+from tests.factories.lti_user import LTIUser
```

Comment:
It turns out that LMS already had `factory-boy` as a dependency but wasn't using it. I've added a `tests.factories` package to provide a place to put shared test factories and establish a pattern.

There's no `factories` pytest fixture like h has. LMS's one factory so far doesn't use the DB so it isn't necessary, and just importing `tests.factories` directly allows them to be used in parametrize.

We might add an h-like `factories` fixture when LMS needs factories for DB models, or figure out a way to do that without requiring the factories to be accessed via a `factories` fixture.

Reply:
Unless we need to I think it's better without, it's kind unnecessarily roundabout if you want to find where it came from.